### PR TITLE
Remove injected CSS/JS that is no longer needed

### PR DESCRIPTION
**Pull request by chipsnyder**
**Fixes** https://github.com/wordpress-mobile/WordPress-Android/issues/13501

### To test:
1. Navigate to the Create WordPress.com site
    - Choose site > ➕ > Create WordPress.com site
1. Choose a Design or Select Skip
1. Search for and choose a domain
1. Select Create Site
1. Wait for your site to be created and for the preview to load
1. **Expect** 
    - To not see a header like the admin bar, the cookie notice or the create a page on WordPress.com notice
    - You also shouldn't be able to click any of the buttons or links on the preview webview. 
        - The menu will open but you shouldn't be able to navigate within there

📓 Just to tie the original PR to this one. This code was originally added here: https://github.com/wordpress-mobile/WordPress-Android/pull/8382

## PR submission checklist:

- [x] I have considered adding unit tests where possible.
- [x] I have considered adding accessibility improvements for my changes.
- [x] I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.

### CODE REVIEW

**Review by malinajirka — COMMENTED** [review]
Thanks @chipsnyder!

The app works as described ;). One thing we might want to do is remove the `hideGetStartedBar` logic from the VM and its tests, wdyt?

Probably unrelated: I noticed I can't even scroll within the webview. I believe it was supported when the site creation flow was introduced. It feels weird specially in the landscape mode as not even the title of the site fits the screen (and I'm not even using a small display). Wdyt?

**Comment by chipsnyder** [comment]
Hey @malinajirka 👋 

> The app works as described ;). One thing we might want to do is remove the hideGetStartedBar logic from the VM and its tests, wdyt?

Oh gosh, I'm a dunce 🤦 Yup I'll take care of that too. I think I was on too much of an autopilot there thanks for pointing this out. 

>Probably unrelated: I noticed I can't even scroll within the webview. 

I noticed this as well, on iOS, you can scroll in that preview so I thought it was odd that there was a difference but each platform matched 16.2 so I wasn't sure what the intended experience was. I agree though I didn't call it out in the description because I felt like it was unrelated. It's a small fix though I'd be happy to explore it here if you don't mind me expanding the scope of this PR.

> It feels weird specially in the landscape mode as not even the title of the site fits the screen (and I'm not even using a small display). Wdyt?

I think the scrolling really depends on if we want users to really investigate their created site or just see "proof" that it did work. But ultimately I don't see a reason to prevent the scroll.

>I believe it was supported when the site creation flow was introduced. 

This helps a bunch. If this used to work this way then I say we enable it. I went ahead and added this to the PR.

File: WordPress/src/main/res/layout/site_creation_preview_web_view_container.xml [hunk 1]
```diff
@@ -42,7 +42,7 @@
             </FrameLayout>
         </com.google.android.material.card.MaterialCardView>
 
-        <org.wordpress.android.ui.WPWebView
```
Comment (chipsnyder) [inline WordPress/src/main/res/layout/site_creation_preview_web_view_container.xml:45]: @malinajirka Here's the change for the scrolling. I'll be honest I didn't do a deep dive into this, I just saw that `NestedWebView` is what we're using here: https://github.com/wordpress-mobile/WordPress-Android/blob/95b15937bb6a342f3cdfb80e8e5e744a0c0aff82/WordPress/src/main/res/layout/home_page_picker_preview_fragment.xml#L44 and it seems like this has the same nested view hierarchy problem mentioned here https://github.com/wordpress-mobile/WordPress-Android/blob/95b15937bb6a342f3cdfb80e8e5e744a0c0aff82/WordPress/src/main/java/org/wordpress/android/widgets/NestedWebView.kt#L14-L22

Basically just calling this out incase your more familiar with this class.

Comment (malinajirka) [reply WordPress/src/main/res/layout/site_creation_preview_web_view_container.xml:45]: I see, nice catch! This bug was most probably introduced when we re-themed/designed AppBar on all screens of the app.

**Comment by malinajirka** [comment]
LGTM, thanks! ;)

> Oh gosh, I'm a dunce 🤦 Yup I'll take care of that too. I think I was on too much of an autopilot there thanks for pointing this out.

No worries at all ;). It happens to all of us.

**Review by malinajirka — APPROVED** [review]
